lib/lita/handlers/reviewme.rb

```diff
@@ -120,7 +121,8 @@ def next_reviewer(room)
       end
 
       def github_comment(reviewer)
-        ":eyes: @#{reviewer}"
+        msg = config.github_comment_template || ":eyes: %s"
```

Comment:
Yeah I was wondering that too.  Didn't see a clear cut way. This message from the docs was enough for me to not want to explicitly set `config.github_comment_template` somewhere: 
https://docs.lita.io/plugin-authoring/configuration/
>Important! All the values in Lita.config are frozen at runtime. Consider them read only and wrap them with accessor methods in your plugins if you need their raw values to be manipulated in some way.

To me, I read it like if a `github_comment_template is specified` use that... otherwise use the default value.  I went ahead and assigned the default to a constant so things are clearer.

You think setting `config.github_comment_template ||= DEFAULT_MSG` would be clearer?

Happy with either. 👍